### DISCUSSION TITLE
New Project builds with error

### DISCUSSION THREAD

**Discussion by gfo2007** [serenity-is/Serenity#5936] [Help] [answered]
Hi,
I just created a new project but I have unable to build it successfully as the DashboardPage.cs has errors.

Below is the image of the error.

![serene error](https://user-images.githubusercontent.com/14317420/130568236-26469634-4118-4d94-a63a-0fd6bb40b0bb.PNG)

Please help.

**Comment by VictorTomaili** [comment, marked as answer]
use [this sample](https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/Common/Dashboard/DashboardPage.cs), it's fixed but waiting for the next release.

**Reply by gfo2007** [reply]
@VictorTomaili thanks for your quick response. It's still building with errors after replacing the code with the one you showed me.

![DashboardPage extra errors](https://user-images.githubusercontent.com/14317420/130569840-64a07a68-938c-4eda-9c9b-85f4a9593e50.PNG)

**Reply by VictorTomaili** [reply]
you didn't install the northwind module, just eliminate northwind-related code.

**Reply by gfo2007** [reply]
@VictorTomaili thank you. I have removed it and it has compiled without errors.

However, even though I unchecked all boxes, Theme Samples are still showing on my Navigation Menu. Is there a way to easily remove it?

**Reply by VictorTomaili** [reply]
Check navigation.cs files in your project. If there is nothing related, check your nuget packages for themesamples

**Reply by gfo2007** [reply]
@VictorTomaili I had to uninstall the Theme Samples nuget package and then remove the code from Startup.cs before it got removed from the Navigation Menu.

I hope all these issues get fixed in the release.

Thank you very much for your time.
